pom.xml

```diff
@@ -286,6 +287,11 @@
         <artifactId>commons-lang3</artifactId>
         <version>${commons-lang3-version}</version>
       </dependency>
+      <dependency>
```

Comment:
not sure what it does to only add this in dependency management. Was a submodule using this without specifying the version somewhere?

Reply:
`commons-collections` is brought in by a transitive dependency, so using dependencyManagement forces that transitive dependency to use a newer version.